src/components/Plugin/SafeSnap/Modal/OptionApproval.vue

```diff
@@ -54,11 +86,23 @@ export default {
           class="question-link"
           rel="noreferrer noopener"
           target="_blank"
-          :href="questionLink"
+          :href="criteriaLink"
```

Comment:
If there is no requirements text record this link should not be shown and the question should be only "Did the proposal pass?"

@samuveth that's maybe the issue you are facing?